app/controllers/GoodsDescriptionController.scala

```diff
@@ -92,11 +93,15 @@ class GoodsDescriptionController @Inject() (
         .bindFromRequest()
         .fold(
           formWithErrors => Future.successful(BadRequest(view(formWithErrors, mode, submitAction))),
-          value =>
+          value => {
+            val oldValue = request.userAnswers.get(GoodsDescriptionUpdatePage(recordId)).getOrElse("")
             for {
               updatedAnswers <- Future.fromTry(request.userAnswers.set(GoodsDescriptionUpdatePage(recordId), value))
               _              <- sessionRepository.set(updatedAnswers)
             } yield Redirect(navigator.nextPage(GoodsDescriptionUpdatePage(recordId), mode, updatedAnswers))
+              .addingToSession("changesMade" -> (oldValue != value).toString)
+              .addingToSession("changedPage" -> "goods description")
```

Comment:
Can we add these pages to the session data constants file too please? :)

Reply:
done!